docs/advanced-ping/pingask.md

```diff
@@ -0,0 +1,66 @@
+# Ping Ask
+
+Ping now offers a new service for your content, Ping Ask!
+
+Ping Ask is a way for your viewers to submit questions and for you to be able to neatly display them on stream, perfect for AMAs, QnAs, and much more!
+
+And in true Ping fashion, its easy and quick to Setup
+
+## Setup
+
+First, we'll head on over the ask.ping.gg and log in
+
+![PingAsk Login page](https://i.imgur.com/b3e1KEC.png)
+
+Authenticate with your log in of choice and you'll be brought to the main screen, it is here that we will grab our browser source link
+
+![copyembedpingask](https://i.imgur.com/PRDZk5X.png)
+
+In OBS, we'll make a browser source. Any resolution will do fine to meet the needs of your overlays and stream setup, for this I'll leave it at 800x600. Plug the Embed URL into the URL box and hit OK
+
+![pingaskbrowsersource](https://i.imgur.com/6olV1SA.png)
+
+## Viewer questions
+
+Viewers will be able to submit questions via the ***Q&A URL***
+
+You can copy the Q&A URL on your dashboard on Ping Ask in the top right corner
+
+![pingaskqnaurl](https://i.imgur.com/saklBdj.png)
+
+This is the link that your audience will click to be able to submit questions! 
+
+:::tip
+
+We reccomend that you make a chat command with your chat bot for viewers to quickly and easily find and click the link.
+
+::: 
+
+![viewerquestion](https://i.imgur.com/0pzqE3H.png)
+
+When viewers submit their questions, they'll show up in your dashboard.
+
+![questionondashboard](https://i.imgur.com/gZSpPpf.png)
+
+When you over over the question, you'll see a prompt to show on stream. Click it and it'll do so!
+
+![shownonobs](https://i.imgur.com/AI5Qxgv.png)
+
+Give your answer to your audience and then click "Next Question" on your dashboard

```

Comment:
Maybe a warning that clicking "Next question" will remove the question from your queue and you will not be able to bring it back.